src/main/kotlin/com/pubnub/api/services/PresenceService.kt

```diff
@@ -31,14 +31,14 @@ internal interface PresenceService {
         @QueryMap options: Map<String, String>
     ): Call<Envelope<WhereNowPayload>>
 
-    @GET("v2/presence/sub_key/{subKey}/channel/{channel}")
+    @GET("v2/presence/sub-key/{subKey}/channel/{channel}")
     fun hereNow(
         @Path("subKey") subKey: String,
         @Path("channel") channel: String,
         @QueryMap options: Map<String, String>
     ): Call<Envelope<JsonElement>>
 
-    @GET("v2/presence/sub_key/{subKey}")
+    @GET("v2/presence/sub-key/{subKey}")
```

Comment:
I surprised. Was this change somehow announced?

Reply:
I see PR from Serhii. I am wondering why openAPI doesn't reflect those changes?

Reply:
Can we make sure it works well? Additionally, can we check all the `sub-key` usage? We found that `sub-key` and `sub_key` are mixed in various SDKs

Reply:
Please make sure that all the SDKs are implementing it the same way. We're using it in Chat Team as well (we'll have to fix the contracts)

Reply:
@Chesteer89 sure. I'll make sure that we have consistent paths at least across SDKS used in chat components before merging this